subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/notes/tasks/UpdateReleaseNotesOnGitHub.java

```diff
@@ -58,24 +53,37 @@ private String findReleaseByTagName(UpdateReleaseNotesOnGitHubTask task) throws
         }
     }
 
-    private void editRelease(String releaseId, String text, UpdateReleaseNotesOnGitHubTask task) throws Exception {
-        JsonObject body = new JsonObject();
-        body.put("body", text);
+    private void createReleaseByTagName(UpdateReleaseNotesOnGitHubTask task, String text) throws Exception {
+        String tagName = tagName(task);
+        String url = "/repos/" + task.getUpstreamRepositoryName() + "/releases";
 
-        String url = "/repos/" + task.getUpstreamRepositoryName() + "/releases/" + releaseId;
-        LOG.debug("GitHub edit release notes on release page PATCH {} body: {}", url, body.toJson());
         if (task.isDryRun()) {
-            LOG.lifecycle("It's -PdryRun mode, releases notes on GitHub will be NOT changed." +
-                "\n  It will execute in normal mode: PATCH {}", url);
+            LOG.lifecycle("It's -PdryRun mode, releases notes on GitHub will be NOT created." +
+                "\n  It will execute in normal mode: POST {}", url);
             return;
         }
-        LOG.lifecycle("PATCH {}", url);
-        String response = gitHubApi.patch(url, body.toJson());
 
-        JsonObject responseJson = (JsonObject) Jsoner.deserialize(response);
+        JsonObject body = new JsonObject();
+        body.put("tag_name", tagName);
+        body.put("name", tagName);
+        body.put("body", text);
 
-        String htmlUrl = responseJson.getString("html_url");
-        LOG.lifecycle("  Successfully updated release notes on GitHub: {}", htmlUrl);
+        LOG.debug("GitHub create release by tag name POST {}", url);
+        LOG.lifecycle("POST {}", url);
```

Comment:
a minor one: we can get rid of one of this log lines since they basically provide the same information.

Reply:
Thanks a lot for the review, I removed duplicated logs in this class (and checked other classes for 2.1.0). Next PRs coming soon...